CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ### Changed
 
+- ([#878](https://github.com/demos-europe/demosplan-ui/pull/878)) BREAKING: Prevent "select all" in DpDataTable to select "locked" items. Before all items per page where selected, independed of the the lock state ([@salisdemos](https://github.com/salisdemos))
```

Comment:
this should go into the "unreleased" section.

Reply:
omg. thanks!

Reply:
d9cc57c50bca011703470b9b5bef024ab53a9e37